databricks-notebooks/splink notebook - databricks-splink.py

```diff
@@ -0,0 +1,97 @@
+# Databricks notebook source
```

Comment:
Do we want to drop this file from the commit? 

I think it's a good idea to have a working example, so I'd like to include this,  but it would fit better in the following repo:
https://github.com/moj-analytical-services/splink_demos

The `.ipynb` files in the repo with the `example_` prefix then make their way into the docs [here](https://moj-analytical-services.github.io/splink/examples_index.html). It would be also possible to commit as a `.md` file with a big codeblock (`example_databricks_notebook.md`), if we wanted the example to just contain the raw code.

Also happy just to drop this file for now, in the interests of getting this merged!

Reply:
lets drop - we can build this out as part of the planning for the January hack day